### PR TITLE
[16.0][FIX] account_invoice_triple_discount: discount decimal precision

### PR DESCRIPTION
**Pull request by Segui-S73**
When calculating the triple discount, if there is any raise the decimal precision of the discount field stays at 16, so try except is added to avoid this.

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
thanks !

**Comment by rafaelbn** [comment]
Please @eantones could you review this PR?

**Comment by Segui-S73** [comment]
Hi @rafaelbn , I have manually the rebase after seeing the bot error.

File: account_invoice_triple_discount/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Account Invoice Triple Discount",
-    "version": "16.0.1.0.1",
+    "version": "16.0.1.0.2",
```
Comment (rafaelbn) [inline account_invoice_triple_discount/__manifest__.py]: You don't need to update version. This is made by bot automatically.

**Review by rafaelbn — APPROVED** [review]

**Pull request closed without merging**